postgres-appliance/tests/test_spilo.sh

```diff
@@ -292,7 +292,9 @@ function test_spilo() {
 
 
     # TEST SUITE 1
-    find_leader "$upgrade_container" # wait clone to finish and prevent timescale installation gets cloned
+    # wait clone to finish and prevent timescale installation gets cloned
+    find_leader "$upgrade_container"
+    find_leader "$upgrade3_container"
```

Comment:
Just curious, so this is to make sure all clones are completed ?

Reply:
yes. before timescale extension creation is called

Reply:
Got it